### PR TITLE
Update advanced-js-bundling.md

### PR DESCRIPTION
**Pull request by chirag-wagento**
## This PR is a:
- [ ] New topic
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement
 

## Summary
 Fixes Typo
When this pull request is merged, it will...

### CODE REVIEW

File: guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ When your application is not in the production mode, you can set Magento to comp
 
 To set the compilation mode, do the following:
 <ol>
-<li>In the Magento Admin, navigate to <b>Stores</b> > <b>Configuration</b> > ADVANCED > <b>Developer</b>.</li>
+<li>In the Magento Admin, navigate to <b>Stores > Configuration > ADVANCED > Developer</b>.</li>
```
Comment (shrielenee) [inline guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md]: We now use markdown formatting for lists. Please replace the HTML with the markdown version of lists: https://daringfireball.net/projects/markdown/syntax#list

Comment (shrielenee) [reply guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md]: We use markdown for text formatting. Please replace `<b>` with `**` surrounding what you want to be bold.

And ensure that you do NOT include the `>` in the bolded text.

File: guides/v2.2/frontend-dev-guide/css-topics/css-preprocess.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ When your application is not in the production mode, you can set Magento to comp
 
 To set the compilation mode, do the following:
 <ol>
-<li>In the Magento Admin, navigate to <b>Stores</b> > <b>Configuration</b> > ADVANCED > <b>Developer</b>.</li>
+<li>In the Magento Admin, navigate to <b>Stores > Configuration > ADVANCED > Developer</b>.</li>
```
Comment (shrielenee) [inline guides/v2.2/frontend-dev-guide/css-topics/css-preprocess.md]: See previous comments.

File: guides/v2.3/performance-best-practices/advanced-js-bundling.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ In a modular application, the number of server requests can reach into the hundr
 ![noBundling](images/noBundling.png)
 
 ### Magento merging and bundling
-Out of the box, Magento provides two ways to reduce the number of server requests: merging and bundling. These settings are turned off by default. You can turn them on within Magento's Admin UI in Stores > Configuration > Advanced > Developer > JavaScript Settings, or from the command line.
+Out of the box, Magento provides two ways to reduce the number of server requests: merging and bundling. These settings are turned off by default. You can turn them on within Magento's Admin UI in **Stores > Configuration > Advanced > Developer > JavaScript Settings**, or from the command line.
```
Comment (shrielenee) [inline guides/v2.3/performance-best-practices/advanced-js-bundling.md]: See previous comments.

File: guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md (hunk 1 above)
Comment (shrielenee) [reply guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md]: You only have to make the change to the `css-preprocess.md` file in v2.1, as 2.2 and 2.3 versions are symlinked to 2.1. Please remove v2.2 and v2.3 changes.

**Comment by chirag-wagento** [comment]
@shrielenee  I have committed with new changes please check it.

Thanks

File: guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md [hunk 4]
```diff
@@ -71,7 +71,7 @@ When your application is not in the production mode, you can set Magento to comp
 
 To set the compilation mode, do the following:
 <ol>
-<li>In the Magento Admin, navigate to <b>Stores</b> > <b>Configuration</b> > ADVANCED > <b>Developer</b>.</li>
+<li>In the Magento Admin, navigate to <b>Stores</b> > <b>Configuration</b> > <b>ADVANCED</b> > <b>Developer</b>.</li>
```
Comment (shrielenee) [inline guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md]: In this page there are multiple HTML-formatted lists. Please convert them to markdown lists: https://daringfireball.net/projects/markdown/syntax#list

File: guides/v2.3/performance-best-practices/advanced-js-bundling.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ In a modular application, the number of server requests can reach into the hundr
 ![noBundling](images/noBundling.png)
 
 ### Magento merging and bundling
-Out of the box, Magento provides two ways to reduce the number of server requests: merging and bundling. These settings are turned off by default. You can turn them on within Magento's Admin UI in Stores > Configuration > Advanced > Developer > JavaScript Settings, or from the command line.
+Out of the box, Magento provides two ways to reduce the number of server requests: merging and bundling. These settings are turned off by default. You can turn them on within Magento's Admin UI in **Stores** > **Configuration** > **ADVANCED** > **Developerv** > **JavaScript Settings**, or from the command line.
```
Comment (shrielenee) [inline guides/v2.3/performance-best-practices/advanced-js-bundling.md]: "Advanced" doesn't need to be capitalized, and you've added an extra "v" to "Developer".

**Comment by chirag-wagento** [comment]
@shrielenee As per your advice I have committed  new changes please check it.

Thanks

File: guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md [hunk 6]
```diff
@@ -58,50 +56,35 @@ The root source files for the Blank theme:
...
+5.  All source files are passed to the {% glossarytooltip bf703ab1-ca4b-48f9-b2b7-16a81fd46e02 %}PHP{% endglossarytooltip %} LESS compiler. The resulting compiled `.css` files are published to `pub/static/frontend/<Vendor>/<theme>/<locale>`.
 
 #### Styles debugging in server-side compilation mode {#css_debug_server}
 
 In server-side LESS compilation mode, to have your changes applied, clear <code>pub/static/frontend/&lt;Vendor&gt;/&lt;theme&gt;/&lt;locale&gt;</code> by deleting the directory in the file system, and reload the store pages to trigger compilation and publication.
 
-<div class="bs-callout bs-callout-info" id="info">
-  <p>You might also need to clear the <code>var/cache</code> and <code>var/view_preprocessed</code> directories.</p>
-</div>
+You might also need to clear the `var/cache` and `var/view_preprocessed` directories.
```
Comment (shrielenee) [inline guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md]: Please revert this - We actually do use HTML for notes or callouts. I apologize for not mentioning this!

File: guides/v2.3/performance-best-practices/advanced-js-bundling.md [hunk 7]
```diff
@@ -20,7 +20,7 @@ In a modular application, the number of server requests can reach into the hundr
 ![noBundling](images/noBundling.png)
 
 ### Magento merging and bundling
-Out of the box, Magento provides two ways to reduce the number of server requests: merging and bundling. These settings are turned off by default. You can turn them on within Magento's Admin UI in Stores > Configuration > Advanced > Developer > JavaScript Settings, or from the command line.
+Out of the box, Magento provides two ways to reduce the number of server requests: merging and bundling. These settings are turned off by default. You can turn them on within Magento's Admin UI in **Stores** > **Configuration** > **Advanced** > **Developer** > **JavaScript Settings**, or from the command line.
```
Comment (shrielenee) [inline guides/v2.3/performance-best-practices/advanced-js-bundling.md]: Good job.

File: guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md [hunk 8]
```diff
@@ -43,11 +43,9 @@ For example, in one of the <a href="https://github.com/magento/magento2/blob/2.0
 
 The root source files for the Blank theme:
 
-<ul>
-<li><code><a href="{{ site.mage2000url }}app/design/frontend/Magento/blank/web/css/styles-m.less" target="_blank">&lt;Magento_Blank_theme_dir&gt;/web/css/styles-m.less</a></code></li>
-<li><code><a href="{{ site.mage2000url }}app/design/frontend/Magento/blank/web/css/styles-l.less" target="_blank">&lt;Magento_Blank_theme_dir&gt;/web/css/styles-l.less</a></code></li>
-<li><code><a href="{{ site.mage2000url }}app/design/frontend/Magento/blank/web/css/print.less" target="_blank">&lt;Magento_Blank_theme_dir&gt;/web/css/print.less</a></code></li>
-</ul>
+*   `[<Magento_Blank_theme_dir>/web/css/styles-m.less]({{ site.mage2000url }}app/design/frontend/Magento/blank/web/css/styles-m.less)`
```
Comment (shrielenee) [inline guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md]: Please also revert this. Markdown code is not supported **within** HTML tables. Within HTML tables you must use HTML code only.

File: guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md [hunk 9]
```diff
@@ -58,50 +56,35 @@ The root source files for the Blank theme:
...
 To set the compilation mode, do the following:
-<ol>
-<li>In the Magento Admin, navigate to <b>Stores</b> > <b>Configuration</b> > ADVANCED > <b>Developer</b>.</li>
-<li>In the <b>Store View</b> drop-down field, select <b>Default Config</b>.</li>
-<li>Under <b>Front-end development workflow</b>, in the <b>Workflow type</b> field, select the compilation mode.</li>
-<li>To save the settings, click <b>Save Config</b>.</li>
 
-<li>Make sure that the same compilation mode is set for each configuration scope. That is, check the <b>Front-end development workflow</b> option having switched the <b>Store View</b> drop-down field to the {% glossarytooltip a3c8f20f-b067-414e-9781-06378c193155 %}website{% endglossarytooltip %} scope first, and then to the {% glossarytooltip ca5a9ff1-8182-4fc4-a34b-9b3f831dbf3f %}store view{% endglossarytooltip %}. Change the option to match the default config if it is different.</li>
-</ol>
+1.  In the Magento Admin, navigate to **Stores** \> **Configuration** \> **ADVANCED** > **Developer**.
```
Comment (shrielenee) [inline guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md]: Change `ADVANCED` to `Advanced`.

File: guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md [hunk 10]
```diff
@@ -63,7 +65,7 @@ In the Magento application, the following modes of compiling <code>.less</code>
 
 To set the compilation mode, do the following:
 
-1.  In the Magento Admin, navigate to **Stores** \> **Configuration** \> **ADVANCED** > **Developer**.
+1.  In the Magento Admin, navigate to **Stores** \> **Configuration** \> **Advanced** > **Developer**.
```
Comment (shrielenee) [inline guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md]: There are two extra `/` here for some reason. I'll remove them, but make sure you check for inconsistencies like this!

File: guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md [hunk 11]
```diff
@@ -63,7 +65,7 @@ In the Magento application, the following modes of compiling <code>.less</code>
...
 2.  In the **Store View** drop-down field, select **Default Config**.
```
Comment (shrielenee) [inline guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md]: Also, "drop-down" should be "dropdown".

**Comment by shrielenee** [comment]
Also remember that to apply code formatting to text use single backticks surrounding the words, like ` `code` ` **unless** it's within an HTML table or note (which requires HTML only).

**Comment by shrielenee** [comment]
I've applied a hefty copyedit and will be merging your changes along with mine. Thanks @chirag-wagento!

**Pull request merged**